.idea/encodings.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Can you remove this file from this PR, since it is not related with your issue and because this is specific for the way you use Android Studio (if I'm not mistaken)

Reply:
@isabelcosta Am i supposed to delete the entire .idea file from this PR or only idea/encodings.xml ?

Reply:
Although i have deleted encodings.xml .